payment/controllers.go

```diff
@@ -309,19 +309,26 @@ func GetOrderCreds(service *Service) handlers.AppHandler {
 
 		id := uuid.Must(uuid.FromString(orderID))
 
-		creds, err := service.datastore.GetOrderCreds(id)
+		creds, err := service.datastore.GetOrderCreds(id, false)
 		if err != nil {
 			return handlers.WrapError(err, "Error getting claim", http.StatusBadRequest)
 		}
 
 		if creds == nil {
 			return &handlers.AppError{
-				Message: "Order does not exist",
+				Message: "Credentials do not exist",
 				Code:    http.StatusNotFound,
 				Data:    map[string]interface{}{},
 			}
 		}
 
+		for i := 0; i < len(*creds); i++ {
+			if (*creds)[i].SignedCreds == nil {
+				w.WriteHeader(http.StatusAccepted)
+				break
+			}
+		}
+
 		w.WriteHeader(http.StatusOK)
```

Comment:
In particular circumstances you will be attempting to WriteHeader twice.  It will ignore the second attempt, but maybe we could check a "sentHeader" bool value or something to make sure we don't double write?  If we used the `RenderContent` helper function we could perform render content directly on 327 and return from that point instead?

Reply:
https://github.com/brave-intl/bat-go/pull/326/commits/d3fa4e7ca74b8f334b0dcd7cbc2c2f75129d4995